### DISCUSSION TITLE
表单设计器 antd 样式问题

### DISCUSSION THREAD

**Discussion by GadflyX** [alibaba/formily#2088] [Q&A 2.x] [answered]
这是一条求助：
      根据 官方文档和 参考 Playground 引入designable
      发现样式全部都没有生效
      import 'antd/dist/antd.less'
      这是我在组件内引入 antd.less 的样子
      也看了一下我现在的 antd 版本和源码的 antd 版本
      对比了一下，发现基本上一致。
      想问下作者大大，知道这是什么原因么？

**Comment by GadflyX** [comment, marked as answer]
已经解决，发现 playground 的样式压根不是来自 antd，误会错了意
在官网 playground 找到了对应的 css

**Reply by manong333** [reply]
老哥 ，你找到对应的css怎么找到的

**Reply by WLyKan** [reply]
建议详细说一说解决问题的过程思路
